components/Account/AccountOverview.tsx

```diff
@@ -143,28 +143,24 @@ const AccountOverview = ({ address }: AccountOverviewProps): JSX.Element => {
                     </Grid>
 
                     <Grid item xs={9}>
-                        {accountQuery?.data?.account?.balance ? (
-                            <Typography variant="h5" className={classes.alignRight}>
-                                {Coin(accountQuery?.data?.account?.balance, 'CELO', 2)}
-                            </Typography>
-                        ) : (
-                            <NotAvailable variant="h5" className={classes.alignRight} />
```

Comment:
@MonikaCat please remove the unused `<NotAvailable />` components

Reply:
Removed 👍🏻